Main.java

```diff
@@ -0,0 +1,25 @@
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        Party vvd = new Party("VVD");
```

Comment:
Op zich is de ze manier van implementeren prima. Het verandert niets aan de logica. De UML gaf echter aan dat er 'hard getypeerde' subclasses zijn. Die zou je kunnen maken door:

`class VVD extends Party` en dan `set` je de naam 'VVD' in de `constructor`.  Je hoeft dit niet aan te passen, omdat het voor de werking dus niet uitmaakt.